src/main/java/sirius/kernel/nls/NLS.java

```diff
@@ -1315,6 +1315,20 @@ private static void appendDurationValue(StringBuilder result, String key, long v
         result.append(get(key, (int) value));
     }
 
+    /**
+     * Converts the given time range in milliseconds to a time representation format typically found in digital clocks.
```

Comment:
```suggestion
     * Converts the given time range in milliseconds to a time representation format typically found in digital clocks (like 12:05:17).
```

Wär schon nett wenn das JavaDoc sagt wie es hinterher aussieht :)